src/__tests__/Autocomplete.test.tsx

```diff
@@ -276,49 +278,50 @@ describe('Autocomplete', () => {
       fireEvent.blur(inputNode)
 
       // wait a tick for blur to finish
-      setTimeout(() => {
-        expect(inputNode.getAttribute('aria-expanded')).not.toBe('true')
-      }, 0)
+      await waitFor(() => expect(inputNode.getAttribute('aria-expanded')).not.toBe('true'))
```

Comment:
This strategy of waiting a tick was actually failing silently because the assertion would fail after the test completes (in a different 'thread'). Introducing async tests exposed this problem because the failure would actually happen during the next test, because async calls get put at the end of the stack. `await waitFor` is what we wanted for here anyway.